upf/exceptions.py

```diff
@@ -24,3 +24,6 @@ class UPFProblemDefinitionError(UPFException):
 
 class UPFTypeError(UPFException, TypeError):
     pass
+
+class UPFUnraisableError(UPFException):
```

Comment:
"There are only 10 hard things in computer science... naming things, binary numbers, and off-by-one errors." (adapted)
```suggestion
class UPFUnreachableCodeError(UPFException):
```